study.md

```diff
@@ -54,5 +54,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+<!-- From the reading I understand that streams are groups of data structures (data arranged in certain format ) which can be infinite; therefore some methods can't be applied in streams; methods that are applied to streams are methods that perform an action on each item.-->
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.